src/procedure_manager.rs

```diff
@@ -125,17 +128,37 @@ async fn process_commands(connection: &Channel<Command>) {
 }
 
 // STDOUT logging
-async fn read_stdout(stdout_buffer: &mut BufReader<ChildStdout>, procedure_name: &String, path: &String, command: &String) {
+async fn read_stdout(stdout_buffer: &mut BufReader<ChildStdout>, procedure_name: &String, path: &String, command: &String, log: &Option<String>) {
+    let log_pattern = match log {
+        Some(log_pattern) => log_pattern,
+        None => return
+    };
     let mut stdout_reader = stdout_buffer.lines();
     while let Some(line) = stdout_reader.next_line().await.unwrap() {
-        info!(format!("[{}] [{}] Command ({}): {}", procedure_name, path, command, line));
+        let out: String = log_pattern.clone()
```

Comment:
Why is this cloned?

Reply:
I don't want to modify the original pattern every time the while loop runs

Reply:
Hmm. I guess this is the best way

Reply:
I have determined that my initial reserves are unwarranted. Fixing.